subprojects/ide/src/main/java/org/gradle/plugins/ide/idea/model/Project.java

```diff
@@ -321,6 +322,11 @@ private Node findOrCreateLibraryTable() {
         return libraryTable;
     }
 
+    private static void setNodeAttribute(Node node, String key, Object value) {
```

Comment:
Extract this to a separate class and remove the duplication.

Reply:
This was actually on purpose to avoid a random one-line utility method that most people dealing with such code will not remember. Given its an oneliner in three places, I inlined it. But no strong opinion. How/where would you put something like that?

Reply:
It's fine like this, I just wanted to point out that it might make sense to change it.